projects/ngx-charts-on-fhir/src/lib/data-layer/data-layer-manager.service.ts

```diff
@@ -1,5 +1,5 @@
 import { forwardRef, Inject, Injectable, NgZone } from '@angular/core';
-import { BehaviorSubject, distinctUntilChanged, map, merge, Observable, ReplaySubject, throttleTime } from 'rxjs';
+import { BehaviorSubject, distinctUntilChanged, map, merge, Observable, of, ReplaySubject, throttleTime } from 'rxjs';
```

Comment:
unused import?

Reply:
Yes ,I am remove all unused import from that file.